src/sandesh/Assignment_8/NumberDivisibilityWhileLoop.java

```diff
@@ -0,0 +1,67 @@
+//WAP of Assignment 6 using While loop.
+
+package sandesh.Assignment_8;
+
+import java.util.Scanner;
+
+public class NumberDivisibilityWhileLoop {
+
+	void getEvenNumbers(int startPoint, int endPoint){
+		System.out.println("Even numbers within range " + startPoint + " and " + endPoint + " are -: ");
+		int index = startPoint;
```

Comment:
You don't require index variable, you can handle with startPoint variable as well

Reply:
Thank you Harsh. Yes, will take care of this as we save memory when not using any extra variable.